### DISCUSSION TITLE
How to remove saved request?

### DISCUSSION THREAD

**Discussion by JaktensTid** [aws/aws-lambda-dotnet#1314] [Q&A] [answered]
Hi,

Is this possible to delete request I saved?

**Comment by ashishdhingra** [comment, marked as answer]
@JaktensTid Good afternoon. Looks like you are referring to saved requests in Lambda Test Tool. From the UI, there doesn't appear to be an option to delete saved requests. Upon investigating further, looks like the requests are saved under `.lambda-test-tool\SavedRequests` directory which is created under the .NET project directory from where the Lambda Test tool is executed. The saved requests are stored in file with name `<<requestname>>.json`. You may manually delete the saved file to remove the request from the list.

Thanks,
Ashish
